api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -193,4 +196,17 @@ public boolean isUserNameAsEmailValid(String username) {
 		Matcher matcher = EMAIL_PATTERN.matcher(username);
 		return matcher.matches();
 	}
+	
+	/**
+	 * @return true if email is valid or null
+	 * @param email
+	 */
+	public boolean isEmailValid(String email) {
```

Comment:
This should probably be private if you don't call it outside of this class

Reply:
Did you see this comment? Please check through for all unaddressed comments

Reply:
I used this method in UserValidatorTest So i needed it to be public

Reply:
That's not the right way to do it, you should be testing the validator that method is internal logic to the validator, what happens inside the validator should be treated like a blackbox. Am saying you should test that if a user has an invalid email, the validator fails and vice versa rather than exposing isEmailValid